src/data/data.JSON

```diff
@@ -14619,5 +14619,166 @@
       ],
       "label": "distribution"
     }
+  },
+  {
+    "company": {
+      "data": "Red Hat",
+      "label": "company"
+    },
+    "system": {
+      "data": "Patternfly Design System",
+      "label": "system",
+      "url": "https://www.patternfly.org/",
+      "$addedAt": "202004221115"
+    },
+    "repository": {
+      "data": "GitHub",
+      "label": "repository",
+      "url": "https://github.com/patternfly/patternfly"
+    },
+    "codeDepth": {
+      "data": "HTML/CSS/JS/TS",
+      "label": "code depth"
+    },
+    "components": {
+      "data": "yes",
+      "label": "components"
+    },
+    "js": {
+      "data": "Patternfly",
+      "label": "JS library/framework",
+      "url": "https://www.patternfly.org/v4/get-started/developers#htmlcss"
+    },
+    "ts": {
+      "data": "yes",
+      "label": "typescript"
+    },
+    "webComponents": {
+      "data": "yes",
+      "label": "web components"
+    },
+    "tests": {
+      "data": [
+        "BackstopJS",
+        "jest",
+        "Web Component Tester"
+      ],
+      "label": "tests"
+    },
+    "linter": {
+      "data": "TSLint",
+      "label": "linter"
+    },
+    "css": {
+      "data": [
+        "Sass",
+        "PostCSS"
+      ],
+      "label": "CSS"
+    },
+    "cssInjs": {
+      "data": "no",
+      "label": "CSS in JS"
+    },
+    "designTokens": {
+      "data": "no",
+      "label": "design tokens"
+    },
+    "bundleManager": {
+      "data": [
+        "Webpack",
+        "Gulp"
+      ],
+      "label": "bundle manager"
+    },
+    "uiKit": {
+      "data": "yes",
```

Comment:
Based on other entries, I believe this is meant to point to the Sketch design kit:
```
  "uiKit": {
    "data": "Sketch",
    "url": "https://github.com/patternfly/patternfly-design-kit",
    "label": "UI kit"
  },
```